resource_processor/shared/config.py

```diff
@@ -4,6 +4,7 @@
 def get_config() -> dict:
     config = {}
 
+    config["azure_location"] = os.environ["RESOURCE_LOCATION"]
```

Comment:
The RP is location agnostic and the API is the one aware of location and sends it in most payloads automatically. 
1. Can you check if this is also true for a shared service template?
2. Is this an intentional change in the pattern?

Reply:
@tamirkamara  No, I didn't realise that the RP is location agnostic and the API is location aware. Don't recall it being mentioned in the docs. I will change this, thanks!